lib/serverengine.rb

```diff
@@ -47,6 +47,8 @@ module ServerEngine
     require File.join(here, v)
   }
 
+  $platformwin = /mswin|mingw/ === RUBY_PLATFORM
```

Comment:
I fixed it.
https://github.com/naritta/serverengine/commit/5bf96ee73de924bb76859a154aa672f9ed823531